src/Bundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sourceability\InstrumentationBundle\Bundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder('sourceability_instrumentation');
+        $rootNode = $treeBuilder->getRootNode();
+
+        $rootNode
+            ->children()
+            ->arrayNode('profilers')
+            ->children()
```

Comment:
It looks like the indentation went away

Reply:
I ran `make cs-fix` and this was the result.

Reply:
Ah, understood, I believe the following has to be updated for `cs-fix` to not mess with the indentation: https://github.com/sourceability/instrumentation-bundle/blob/a877f6cb527c88d9d2121c87a202c104f35f3531/ecs.php#L31-L36

Reply:
Done.